spec/requests/sessions_controller_spec.rb

```diff
@@ -40,8 +40,8 @@
 
   describe 'Account locked after 9 un-successful logins' do
     before(:each) do
-      # FIXME: this should be (Devise.maximum_attempts -1)  (not a hard coded 9, see bug above)
-      9.times do
+      # Devise.maximum_attempts is twice what it should be (see Devise 2FA bug above)
+      ((Devise.maximum_attempts / 2) - 1).times do
```

Comment:
Changing this from hard-coded `9` because it had an implicit dependency on https://github.com/greenriver/hmis-warehouse/blob/5aaf4bf38427712c891f352aa9172c071b52993f/.env.test#L19, which might be overwritten locally by `.env.local` (mine was)